docs/se/grpc/server.adoc

```diff
@@ -513,7 +506,7 @@ While global metrics capture is certainly useful, it is not always sufficient.
 Keeping a separate `timer` for each gRPC method may be an overkill, so the user
 could decide to use a lighter-weight metric type, such as `counter` or a `meter`.
 
-However, she may still want to enable `histogram` or a `timer` for some services,
+However, user may still want to enable `histogram` or a `timer` for some services,
```

Comment:
IMO "she" is OK here, but if we change it to "user" it should be "the user"

Above line 379: "provide a built-in support" -> "provide built-in support" (remove "a")
Line 424: "define custom health check" -> "define custom health checks"
425-426: "using service configurer inside the
`GrpcRouting` deefinition:" -> "using a service configurer inside the `GrpcRouting` definition" (insert "a", fix typo in "deefinition"
442: "using custom implementation" -> "using a custom implementation" (insert "a")
445: "via standard" "via the standard"
470: "via `/health` REST endpoint," -> "via the `/health` REST endpoint,"
474: "Helidon gRPC Server" -> "The Helidon gRPC server"
479: " gRPC Server" -> "the gRPC server"
480: Add a space after the period at the end of the sentence and "provide aggregate view" -> "provide an aggregate view"
483: "fine grained" -> "fine-grained"
506: "an overkill" -> "excessive"
507: "such as `counter`" -> "such as a `counter`"
509: "enable" -> "enable a"
513: "either service" -> "either the service"